src/components/Editor/index.js

```diff
@@ -417,6 +418,25 @@ class Editor extends PureComponent {
     });
   }
 
+  onClick(e: MouseEvent) {
+    const { selectedLocation, jumpToMappedLocation } = this.props;
+
+    if (e.metaKey && e.altKey) {
+      const { line, ch } = this.state.editor.codeMirror.coordsChar({
+        left: e.clientX,
+        top: e.clientY
+      });
+
+      const sourceLocation = {
+        sourceId: selectedLocation.sourceId,
+        line: line + 1,
+        column: ch + 1
+      };
+
+      jumpToMappedLocation(sourceLocation);
```

Comment:
looks like meta should map to windows

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/metaKey